### ISSUE TITLE
Tests not passing for latest nightlies

### ISSUE THREAD

**Issue by oscarlevin** [PreTeXtBook/pretext-cli#665]
Nightly builds have not been passing tests, specifically with respect to the custom webwork server.  

Also 
- [x] Check why I'm not getting notified when actions fail.

**Comment by oscarlevin** [comment]
@Alex-Jordan and @bnmnetp, the reason for our failing tests is that we try to build webwork using webwork.runestone.academy and we get an SSL error: 
```
HTTPSConnectionPool(host='webwork.runestone.academy', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1002)')))
```
I tried to navigate to the webwork server and got a similarly ominous warning about security.

I don't know if there is an easy way for this to be fixed on your end, or if you have a suggestion for another webwork server for us to test against.

**Comment by bnmnetp** [comment]
@Alex-Jordan -- Did the certificate expire?

**Comment by Alex-Jordan** [comment]
Oh, I've only been monitoring `webwork-hosting.runestone.academy`, which I updated a few weeks ago. Sorry! I will update `webwork.runestone.academy` today.

**Comment by Alex-Jordan** [comment]
I just updated the certificate and it passes now at a checker. Sorry about this. Maybe at a drop-in someone can help me set this up to run automatically. It seemed easier with `webwork.runestone.academy` than it is with `webwork-hosting.runestone.academy`. On the latter one, `certbot renew` messes up the permissions of the new certificates and I have to chase those down and fix them. But for whatever reason I did not need to do that step just now with `webwork.runestone.academy`.
